### DISCUSSION TITLE
How could I remove a plotNFT?

### DISCUSSION THREAD

**Discussion by stone10104** [Chia-Network/chia-blockchain#7410] [Support]
I create 2 plot NFTs, and now I want to remove one. How could I do it?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
